win32ss/user/ntuser/hook.c

```diff
@@ -1677,22 +1675,22 @@ NtUserSetWindowsHookEx( HINSTANCE Mod,
        Hook->offPfn = 0;
 
     TRACE("Installing: HookId %d Global %s\n", HookId, !ThreadId ? "TRUE" : "FALSE");
-    RETURN( Handle);
+    Ret = Handle;
 
-CLEANUP:
+Cleanup:
```

Comment:
or `Exit:` as you chose to use in the other functions?
(Note: first two modified functions in message.c also use "Cleanup" label)

Reply:
I use Cleanup when there is something to clean up other than `UserLeave`.

Reply:
OK ! :+1: